.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,84 @@
+name: Build and Deploy to GKE
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  PROJECT_ID: ${{ secrets.PROJECT_ID }}
+  GKE_CLUSTER: pmc-server-prod
+  GKE_ZONE: us-west1
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: production
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      
+    - name: Create .config folder
+      run: mkdir .config
+      
+    - name: create-firebase-secret-json
+      id: create-json-1
+      uses: jsdaniell/create-json@v1.2.2
+      with:
+        name: "firebase_secret_key.json"
+        json: ${{ secrets.FIREBASE_AUTH }}
```

Comment:
Just curious, is secrets a special variable? Or is it like an environment path?

Reply:
It's a secret variable. I added them onto GitHub. Should be in the settings

Reply:
Ah okay! Do you mind uploading some pictures of you testing the changes? Otherwise, everything looks good to me